File: cpp/No_time_to_wait.cpp
#include<iostream>
using namespace std;
int main()
{
    int N, H, x;
    int req;
    scanf("%d %d %d",&N,&H,&x);
    int T[N];
    for (int i=0; i<N;i++)
    {
        cin>>T[i];
    }
    req = H - x; 
    for(int i=0; i<N;i++)
    {
        if (T[i] >= req)
        {
            printf("YES\n");
        }
    }
}

File: python/No_time_to_wait.py
N=int(input())
H=int(input())
x=int(input())
req= H-x
check=0
T= list(map(int,input().split()))
for i in T:
    if (i>=reeq):
        print("YES")
        check=1
if (check == 0):
    print("NO")
